systemverilog-plugin/UhdmAst.cc

```diff
@@ -2570,7 +2570,13 @@ void UhdmAst::process_operation()
         case vpiPowerOp:
             current_node->type = AST::AST_POW;
             break;
-        case vpiPostIncOp: // TODO: Make this an actual post-increment op (currently it's a pre-increment)
+        case vpiPostIncOp: {
+            // TODO: Make this an actual post-increment op (currently it's a pre-increment)
+            const uhdm_handle *const handle = (const uhdm_handle *)obj_h;
+            const UHDM::BaseClass *const object = (const UHDM::BaseClass *)handle->object;
```

Comment:
If we want `object` here for warning details, let's just pass it from `process_object` (like `process_event_control()`) and avoid code duplicaton.